bert-text-classifier/examples/test.html

```diff
@@ -1,14 +1,44 @@
 <meta charset="utf-8"/>
 <html>
-    <head>
-        <title>text classifier</title>
-        <script src="/dist/src/max.sentimentclass.js"></script>
-        <script>
-        textSentimentClassifier.encode("i like strawberries").then(ids => console.log(ids));
-        textSentimentClassifier.predict("i like strawberries").then(res => console.log(res));
-        </script>
-    </head>
-    <body>
-
-    </body>
+  <head>
+  <title>text classifier</title>
+  </head>
+  <body onload=loadModel()>
+    <h1>Text Sentiment Classifier</h1>
+    <div id="div1">
+      <input type="text" id="text_input" name="text_input" value="Enter a sentence.">
+      <input type="button" id="submit_text" value="Analyze">
+    </div>
+    <p>
+      <label for="status">status:</label>
+      <output name="status" id="status">Loading Model...</output>
+    </p>
+  </body>
+  <script src="file://///wsl$/Ubuntu/home/tedchang/pub_git/max-tfjs-models/bert-text-classifier/dist/src/max.sentimentclass.js"></script>
```

Comment:
hi, @tedhtchang should this be updated to a relative path or point to location someone running the examples can access?

Reply:
Thanks for catching this !

Reply:
👍